playbooks/platform-uninstall.yml

```diff
@@ -0,0 +1,123 @@
+---
+#### This playbook will uninstall ACS
+
+ - hosts: database
+   tasks:
+     - name: Stopping the database
+       ansible.builtin.service:
+         name: postgresql{% if ansible_os_family != 'Debian' %}{{ dependencies_version.postgres_major_version }}{% endif %}
+         state: stopped
+         enabled: no
+
+     - name: Removing the Packages   
+       ansible.builtin.package:
+         name: postgresql
+         state: absent
```

Comment:
Please use  `autoremove: yes` (`ansible.builtin.package` is just a wrapper for apt, dnf & yum  modules, so every option that's valid for all package managers is valid.